tests/libpd_test.c

```diff
@@ -798,7 +801,9 @@ void test_1(void)
 	int rtn, oserr;
 	int test_sock, dup_sock;
 	libpd_mq_t test_queue;
+#ifdef NANOMSG_LEGACY
```

Comment:
Is this NANOMSG_LEGACY check required when we are replacing old nanomsg with NNG? Can we not just remove this code under ifdef?